src/App.jsx

```diff
@@ -1,104 +1,23 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
```

Comment:
Consider adding a check to handle cases where `userId` might be undefined or null. This can prevent potential runtime errors when `userId` is not provided or is invalid.